db/migrate/20160310132620_move_loc_id_to_own_model.rb

```diff
@@ -0,0 +1,53 @@
+class MoveLocIdToOwnModel < MigrationWithData
+  def up
```

Comment:
What do we do with duplicate legacy locids? Re-analyze the ontology? @tillmo: What do you think?